packages/x-components/src/components/modals/open-x-modal.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <BaseEventsModalOpen v-on="$listeners" class="x-modal" :opening-event="openingEvent">
```

Comment:
I think we are using `camelCase` for the `props`. I think webstorm's autocomplete uses the `kebab-case` approach. Can you refactor them?

```suggestion
  <BaseEventsModalOpen v-on="$listeners" class="x-modal" :openingEvent="openingEvent">
```